### PR TITLE
api.md Edits!

### PR DESCRIPTION
**Pull request by esmail-elmoussel**

### CODE REVIEW

File: api.md [hunk 1]
```diff
@@ -81,15 +85,23 @@
 
 **Response:**
 
+
 - **200:**
 
 ```json
 {
-  "msg": "Image updated successfully !",
```
Comment (dvmhmdsd) [inline api.md:88]: Why did u remove this ?

File: api.md [hunk 2]
```diff
@@ -116,11 +130,18 @@
 
 ```json
 {
-  "msg": "Phone updated successfully !",
```
Comment (dvmhmdsd) [inline api.md:119]: Same here !

File: api.md [hunk 3]
```diff
@@ -142,15 +165,15 @@
 ```json
 [
   {
-    "id": "1",
+    "_id": "1",
     "title": "Announcement 1",
     "body": "Fake data for announcement ",
     "date": "26-04-2020",
     "type": "operation", // or 'general'/'technical'
```
Comment (dvmhmdsd) [inline api.md:172]: Why did u remove the comment ?

**Review by dvmhmdsd — CHANGES_REQUESTED** [review]

File: api.md (hunk 1 above)
Comment (esmail-elmoussel) [reply api.md:88]: 200 means its OK its done, i think no need for this extra line.

Comment (dvmhmdsd) [reply api.md:88]: This will be shown as it's

Comment (dvmhmdsd) [reply api.md:88]: The front will show the `msg`  not a text as it's

File: api.md (hunk 3 above)
Comment (esmail-elmoussel) [reply api.md:172]: Didn't ,, its just github colors.

File: api.md (hunk 1 above)
Comment (esmail-elmoussel) [reply api.md:88]: + i already sent you the new image , it's obvious really!

Comment (dvmhmdsd) [reply api.md:88]: ok

**Pull request merged**